heroic-core/src/main/java/com/spotify/heroic/metric/MetricManagerModule.java

```diff
@@ -106,32 +101,7 @@
     /**
      * Threshold for defining a "small" query, measured in pre-aggregation sample size
      */
-    private final long smallQueryThreshold;
-
-    /**
-     * @See com.spotify.heroic.metric.consts.ApiQueryConsts#DEFAULT_MUTATE_RPC_TIMEOUT_MS
-     */
-    private final int mutateRpcTimeoutMs;
-
-    /**
-     * @See com.spotify.heroic.metric.consts.ApiQueryConsts#DEFAULT_READ_ROWS_RPC_TIMEOUT_MS
-     */
-    private final int readRowsRpcTimeoutMs;
-
-    /**
-     * @See com.spotify.heroic.metric.consts.ApiQueryConsts#DEFAULT_SHORT_RPC_TIMEOUT_MS
-     */
-    private final int shortRpcTimeoutMs;
-
-    /**
-     * @See com.spotify.heroic.metric.consts.ApiQueryConsts#DEFAULT_MAX_SCAN_TIMEOUT_RETRIES
-     */
-    private final int maxScanTimeoutRetries;
-
-    /**
-     * @See com.spotify.heroic.metric.consts.ApiQueryConsts#DEFAULT_MAX_ELAPSED_BACKOFF_MILLIS
-     */
-    private final int maxElapsedBackoffMs;
+    private final long smallQueryThreshold;    private final MetricsConnectionSettings connectionSettings;
```

Comment:
any idea what it's on one line?

Reply:
Fat fingers? :) 
I will de-fat finger it.